Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM gcc:latest
+
+RUN apt-get update && apt-get install -y \
+    cmake make mold ccache \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /rawhash2
+COPY . /rawhash2
+
+RUN mkdir build && cd build \
+    && cmake .. \
+    && make -j 1
+
+ENTRYPOINT ["./build/bin/rawhash2_usinglib"]

```

Comment:
I changed this to `rawhash2_usinglib` for now to make the CI run properly. Generally, do we want to include build commands and entrypoint like this in the docker, or in workflows description? In any case, I suppose we should finalise on the target name that is used in the workflows.